jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -200,31 +223,33 @@ pipeline {
 
             script {
                 // Jenkins Summary
-                clusterStatus = ''
+                CommunityBuildStatus = ''
                 if ( currentBuild.currentResult == 'SUCCESS' ) {
                     MESSAGE = "CORTX Community Deploy is Success for the build ${build_id}"
                     ICON = 'accept.gif'
                     STATUS = 'SUCCESS'
-            } else if ( currentBuild.currentResult == 'FAILURE' ) {
+                } else if ( currentBuild.currentResult == 'FAILURE' ) {
                     manager.buildFailure()
                     MESSAGE = "CORTX Community Deploy is Failed for the build ${build_id}"
                     ICON = 'error.gif'
                     STATUS = 'FAILURE'
-            } else {
+                } else {
                     manager.buildUnstable()
                     MESSAGE = 'CORTX Community Deploy Setup is Unstable'
                     ICON = 'warning.gif'
                     STATUS = 'UNSTABLE'
                 }
 
+                clusterStatusHTML  = "<pre>${CommunityBuildStatus}</pre>"
+
                 // Email Notification
                 env.build_stage = "${build_stage}"
-                env.cluster_status = "${clusterStatusHTML}"
+                env.cluster_status = "${clusterStatusHTML }"
```

Comment:
Pls remove the extra space before closing curly bracket

Reply:
Fixed please review again.